ccd-definition/CaseEvent/User/UserEventsCUI.json

```diff
@@ -17,5 +17,23 @@
     "RetriesTimeoutAboutToStartEvent": 0,
```

Comment:
please rename this file to UserEventsCUI-nonprod.json

Reply:
File renamed